### DISCUSSION TITLE
More examples in stdlib Doc

### DISCUSSION THREAD

**Discussion by LukyGuyLucky** [vlang/v#21950] [General]
It would be great for beginners to learn V if the official stdlib Doc provides more examples.

**Comment by jorgeluismireles** [comment]
Folder `v/examples` contains 273 examples alredy:
```
~/v/examples$ v version
V 0.4.6 4c30d35
~/v/examples$ find . -type f -name "*.v" | wc -l
273
```

**Reply by JalonSolov** [reply]
Some of the examples consist of multiple .v files, so it's not quite 273 different examples... but there are, indeed, a lot of them.

You're also a bit behind the current V: `V 0.4.7 bf04adc`.

Besides the examples dir, there are also all the test files, which show how to do things by testing that they work correctly... almost 2_000 files.
